code/__DEFINES/machinery.dm

```diff
@@ -142,15 +142,11 @@ var/list/restricted_camera_networks = list(NETWORK_ERT,NETWORK_MERCENARY,"Secret
 #define MAX_SIPHON_FLOWRATE   2500
 /// L/s. Max flow rate when scrubbing from a turf.
 #define MAX_SCRUBBER_FLOWRATE 200
-// These balance how easy or hard it is to create huge pressure gradients with pumps and filters.
-// Lower values means it takes longer to create large pressures differences.
-// Has no effect on pumping gasses from high pressure to low, only from low to high.
-#define ATMOS_PUMP_EFFICIENCY   2.5
-#define ATMOS_FILTER_EFFICIENCY 2.5
 
 // Will not bother pumping or filtering if the gas source as fewer than this amount of moles, to help with performance.
-#define MINIMUM_MOLES_TO_PUMP		0.01
-#define MINIMUM_MOLES_TO_FILTER		0.01
+#define MINIMUM_MOLES_TO_PUMP		0.001
```

Comment:
latter would be a good idea, though keep in mind; there's a large limit between quantize threshold and this, though it's still good to have a long term plan since pipenets getting just littered with non-used gases is kind of shitty.